src/libs/capture.c

```diff
@@ -416,8 +418,10 @@ int us_capture_hwbuf_grab(us_capture_s *cap, us_capture_hwbuf_s **hw) {
 	_v4l2_buffer_copy(&buf, &(*hw)->buf);
 	(*hw)->raw.grab_ts = (ldf)((buf.timestamp.tv_sec * (u64)1000) + (buf.timestamp.tv_usec / 1000)) / 1000;
 
-	_LOG_DEBUG("Grabbed HW buffer=%u: bytesused=%u, grab_ts=%.3Lf, latency=%.3Lf, skipped=%u",
-		buf.index, buf.bytesused, (*hw)->raw.grab_ts, us_get_now_monotonic() - (*hw)->raw.grab_ts, skipped);
+	_LOG_DEBUG("Grabbed HW buffer=%u: bytesused=%u, length=%u, grab_ts=%.3Lf, latency=%.3Lf, skipped=%u, "
+		"width=%u, height=%u, stride=%u",
+		buf.index, buf.bytesused, buf.length, (*hw)->raw.grab_ts, us_get_now_monotonic() - (*hw)->raw.grab_ts, skipped,
+		(*hw)->raw.width, (*hw)->raw.height, (*hw)->raw.stride);
```

Comment:
Please revert this, there is no point to log resolution because it was reported earlier on configuration stage and didn't changing while working.